src/module.c

```diff
@@ -11142,6 +11068,105 @@ int moduleUnload(sds name) {
     return C_OK;
 }
 
+int moduleConfigsWereSet(RedisModule *module) {
+    listIter li;
+    listNode *ln;
+    listRewind(module->module_configs, &li);
+    while ((ln = listNext(&li))) {
+        ModuleConfig *module_config = listNodeValue(ln);
+        if (!module_config->is_initialized) {
+            return 0;
+        }
+    }
+    return 1;
+}
+
+/* Load a module and initialize it. On success C_OK is returned, otherwise
+ * C_ERR is returned. */
+int moduleLoad(const char *path, void **module_argv, int module_argc, int is_loadex) {
+    int (*onload)(void *, void **, int);
+    void *handle;
+
+    struct stat st;
+    if (stat(path, &st) == 0)
+    {   // this check is best effort
```

Comment:
btw, the one thing that is almost consistent in redis, is that we avoid line comments (`//`).
(the other is that the opening curly brace is in the same line of the `if`, unless it's a multi-line condition)